### ISSUE TITLE
UI for password change/reset missing dashboard

### ISSUE THREAD

**Issue by Ignoramuss** [Ignoramuss/LDERP#2]
The left nav bar and top nav bar of the dashboard missing for password change/reset

**Comment by shvbsle** [comment]
Won't look good. Too many distractions.

**Comment by Ignoramuss** [comment]
I don't think it will look cluttered. As of now it looks too empty. Plus there is no way to go back to the dashboard from any of those pages. It will look more consistent that way IMO. Still, your call

**Comment by shvbsle** [comment]
We'll add a button to go back to the dashboard. Adding a dashboard UI here
seems weird. Consider these points:

-Think about it this way: one moment you are resetting your password and
suddenly you click on "add student". What happens to the password form
then? Get it? Feels weird to leave the form hanging right in the middle of
the password change and switch to another form.
-Now, see the attached image. This form only allows you to change password
completely or cancel it. No third option. This doesn't cause the problem
mentioned previously.
-Also, you have provided a " forgot password" link on home screen. It would
look super weird if it were to take you to a dashboard with no
functionalities but only forgot password form.

**Comment by Ignoramuss** [comment]
Okay. I agree with most of it. Let's settle at a button for going back
